data/data/vsphere/main.tf

```diff
@@ -25,10 +32,20 @@ data "vsphere_network" "network" {
 }
 
 data "vsphere_virtual_machine" "template" {
-  name          = var.vsphere_template
+  name          = vsphereprivate_import_ova.import.name
```

Comment:
what's with the `var.vsphere_template` now? is this no longer required so maybe we drop it.

Reply:
Thanks @abhinavdahiya caught an issue that I missed.  Yeah we still need vsphere_template since that holds what the name of the template will be.  I shouldn't have set the name of the import_ova and instead should have used the variable.  That has been fixed.